specification/columns/chargecategory.md

```diff
@@ -1,7 +1,6 @@
 # Charge Category
 
-A Charge Category indicates whether the row represents an upfront or recurring fee, cost of usage that already occurred,
-an after-the-fact [*adjustment*](#glossary:adjustment) (e.g., credits), or taxes. The Charge Category is commonly used to identify prepaid purchases
+A Charge Category indicates whether the row represents upfront or recurring fee purchases, cost of usage that already occurred or taxes. The Charge Category is commonly used to identify prepaid purchases
```

Comment:
Should we genericize this so the definition doesn't explicitly call out the values? Might be good to define this as the highest-level type of charge the row represents from a billing perspective.

Reply:
Updated to make this more general

Reply:
Please see latest commit for new more generalised wording